pipelineruns/build_pipelinerun_pass.yaml

```diff
@@ -4,6 +4,7 @@ kind: PipelineRun
 metadata:
   generateName: build-pipelinerun-
   annotations:
+    pipelinesascode.tekton.dev/installation-id: '43840620'
```

Comment:
Same here, I don't think we want to add the installation-id to this pipeline, it's supposed to represent a default build pipeline without PaC or PR reporting.

Reply:
removed